.cargo/config.toml

```diff
@@ -1,6 +1,15 @@
 [build]
 
 [env]
+# Stack backtraces.
+RUST_BACKTRACE = "full"
+# Set output levels for `tracing` logging.
+RUST_LOG = "debug,dfir_rs=trace"
+# Make sure all snapshots are written instead of just the first failure.
+INSTA_FORCE_PASS = "1"
```

Comment:
I think we can drop the insta related ones now that Rust Analyzer has native support for running + updating snapshots? `cargo insta` sets these variables.

Reply:
https://insta.rs/docs/cli/

Specifically I've been using `cargo insta test` which skips over failing snapshot tests automatically. Then `cargo insta review` to approve them.

Reply:
Ok I'm going to leave these in `.vscode/settings.json` and not move them to `.cargo/config.toml` so that the VScode run UI still works as before. I tried setting `"rust-analyzer.runnables.command": "cargo insta"` to use the insta harness which works for tests, but breaks running of non tests as `cargo insta run` does not exist